core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1032,10 +1105,19 @@ public long createFile(AlluxioURI path, CreateFileOptions options)
       BlockInfoException, IOException, FileDoesNotExistException {
     Metrics.CREATE_FILES_OPS.inc();
     try (JournalContext journalContext = createJournalContext();
-        LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.WRITE)) {
-      mPermissionChecker.checkParentPermission(Mode.Bits.WRITE, inodePath);
+        LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.WRITE);
+        MasterAuditContext auditContext =
+             createAuditContext("createFile", path, null,
+                 inodePath.peekInode(options.isRecursive()))) {
```

Comment:
If success, we use the target inode.
If failure, HDFS does not print log entry. Since we do, we have the following rules:
1. If recursive, then print the info of the last existing inode in the inode path.
2. If non-recursive, then we should print the information of the parent of the target inode. Basically in this case, we want to audit log to show permission as null.